acceptance/bundle/deploy/dashboard/.gitattributes

```diff
@@ -0,0 +1,2 @@
+# treat new lines in json files in windows as \n
```

Comment:
Correct me if I'm wrong, I thought this issue only showed up when you replace manual mocks in test.toml with testserver implementation but we've decided not to do that for now, right? So it should work without .gitattributes?

Reply:
removed in [0947f16](https://github.com/databricks/cli/pull/3006/commits/0947f1608b82fee17609b1b21e685ddd2d39a700)